src/microbe_stage/editor/ToleranceInfo.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Godot;
+
+/// <summary>
+///   Displays additional tolerance slider info, such as the optimal value and value labels.
+/// </summary>
+public partial class ToleranceInfo : Control
+{
+#pragma warning disable CA2213
+    [Export]
+    private Label startValue = null!;
+
+    [Export]
+    private Label endValue = null!;
+
+    [Export]
+    private TextureRect optimalValueMarker = null!;
+
+    [Export]
+    private Texture2D markerTexture = null!;
+#pragma warning restore CA2213
+
+    private bool showMarker = true;
+
+    public bool ShowMarker
+    {
+        get => showMarker;
+        set
+        {
+            if (value == showMarker)
+                return;
+
+            showMarker = value;
+
+            optimalValueMarker.Visible = value;
+        }
+    }
+
+    public override void _Ready()
+    {
+        base._Ready();
+
+        optimalValueMarker.Texture = markerTexture;
```

Comment:
If this cannot be changed later, then why isn't this directly just setup in the editor to be in the scene directly? Seems a bit weird to have to setup a separate export variable just to set a texture that from what I've seen of the code here so far could very well ahve been done directly in Godot.

Reply:
Okay, I read the main scene file. Maybe this could instead be named "overrideMarkerTexture"? And only apply it if it isn't null?

Also as that's meant to be changed when this scene is placed, I think you should sort out the imports like this:

```c#
    [Export]
    [ExportCategory("Visuals")]
    private Texture2D markerTexture = null!;

    [Export]
    [ExportCategory("Internal")]
    private Label startValue = null!;

    [Export]
    private Label endValue = null!;

    [Export]
    private TextureRect optimalValueMarker = null!;
```

To make it clear what scenes that place this scene are supposed to edit and what not.